lib/utils/stylelintCommand.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const { assertString } = require('./validateTypes');
+
+const DISABLE_COMMAND = 'stylelint-disable';
+const ENABLE_COMMAND = 'stylelint-enable';
+const DISABLE_LINE_COMMAND = 'stylelint-disable-line';
+const DISABLE_NEXT_LINE_COMMAND = 'stylelint-disable-next-line';
+
+const ALL_COMMANDS = new Set([
+	DISABLE_COMMAND,
+	ENABLE_COMMAND,
+	DISABLE_LINE_COMMAND,
+	DISABLE_NEXT_LINE_COMMAND,
+]);
+
+/** @typedef {import('postcss').Comment} Comment */
+
+/**
+ * Extract a command from a given comment.
+ *
+ * @param {Comment} comment
+ * @returns {string}
+ */
+function extractStylelintCommand(comment) {
+	const [command] = comment.text.split(/\s/, 1);
+
+	assertString(command);
```

Comment:
[note] Since `''.split(/\s/, 1)` returns `['']`, this `assertString` should prevent `command` is `undefined`.